@here/harp-examples/example-browser.ts

```diff
@@ -86,12 +152,16 @@ function exampleBrowser(exampleDefinitions: ExampleDefinitions) {
      * @param pageUrl example page url, must exist in `examples` map
      */
     function showExample(pageUrl: string) {
+        const expandButton = document.getElementById("hamburgerMenu") as HTMLAnchorElement;
+        if (navPanel.classList.contains("collapsed")) {
+            expandButton.classList.remove("expanded");
+        } else {
+            expandButton.classList.add("expanded");
+        }
```

Comment:
You can use ```.classList.toggle()``` here

Reply:
As far as I understand, the if condition prevents using toggle, as we want to remove or add depending on it, while we wouldn't control it with toggle.

Reply:
You can pass second argument to toggle
```js
expandButton.classList.toggle("expanded", !navPanel.classList.contains("collapsed"));
```